skycoin/skywire
# improve `skywire-cli vpn` subcommand

Did you run `make format && make check`? Yes

Fixes #1360 

 Changes:	
- [x] improve `skywire-cli vpn start` output.
- [x] fix fetching vpn server list issue.
- [x] improve getRouteSetupHooks (aka autotransport) logic.

How to test this PR:
- for `skywire-cli vpn start <pk>` command, check logs.
- for `vpn list` command, on prod visor check `skywire-cli vpn list --nofilter` command
